### PR TITLE
[ko-KR] Preface translation

### PR DESCRIPTION
**Pull request by EatChangmyeong**
I've translated the preface chapter into Korean (ko-KR); I'd also like to contribute to a few more chapters in the future.

A number of translation concerns:

- The term "Beziér curve" is typically translated into either "베지에 곡선" (~22,800 results in Google Search, with a "Did you mean 베지어 곡선?" suggestion) or "베지어 곡선" (~56,300 results); the former seems to be a more "correct" transliteration (Korean Wikipedia chose "[베지에 곡선](https://ko.wikipedia.org/wiki/%EB%B2%A0%EC%A7%80%EC%97%90_%EA%B3%A1%EC%84%A0)"), but the latter is more common. I decided to go with the former, but I can change this any time.
- The font I'm using here is [Noto Sans Korean](https://fonts.google.com/noto/specimen/Noto+Sans+KR), more specifically a WOFF version in [sangwoobae/noto-sans-korean-webfont](https://github.com/sangwoobae/noto-sans-korean-webfont). The font itself is licensed under OFL, and the repo is under Apache License 2.0.
- `font-size` is 17.2px for now but I'm not confident with this value.
- `locale-strings.js` is fully translated, with the exception of `donationCTA`, `bitcoinHeading` and `bitcoinCTA`.

PS: Thank you for the great work! I had to work with Beziér maths time to time and your primer always helped me a lot.

### CODE REVIEW

**Comment by Pomax** [comment]
Thank you so much for getting Korean translations started! If this is specifically the korean version of noto sans, let's make sure the filename indicates that too so it won't conflict with a future use of another language/locale-specific version of noto, though.

**Review by Pomax — CHANGES_REQUESTED** [review]
let's make sure that font indicates its specific to Korean in its font name, but otherwise this can pretty much go right in!

File: src/html/fragments/fonts.html [hunk 1]
```diff
@@ -32,6 +32,18 @@
         font-size: 17.4px;
     }
 
+{% elif locale == "ko-KR" %}
+
+    @font-face {
+        font-family: "Noto Sans Korean";
+        src: url(./fonts/NotoSans-Regular.woff) format("WOFF");
```
Comment (Pomax) [inline src/html/fragments/fonts.html:39]: let's make sure to rename this to include the "ko" indicator

Comment (EatChangmyeong) [reply src/html/fragments/fonts.html]: I just renamed `NotoSans-Regular.woff` to `noto-sans-korean-regular.woff` and updated the corresponding CSS, I hope this is enough.

Comment (Pomax) [reply src/html/fragments/fonts.html]: yep, that'll do it, in it goes!

**Comment by Pomax** [comment]
Hm, it looks like this might need a rebase, too, since the master branch got updated a few times.

**Comment by EatChangmyeong** [comment]
Actually I'm not used to how Git and GitHub works, but I'll try -- I'll let you know when I succeed.

**Comment by EatChangmyeong** [comment]
I just pulled your changes, do I need to run `npm start` again?

**Comment by Pomax** [comment]
generally yes, that'll generate updated .html files

**Comment by EatChangmyeong** [comment]
All done, is there something else I should do?

File: docs/news/2020-09-18.html [hunk 2]
```diff
@@ -141,15 +149,15 @@ <h5 class="post-date">Fri, 18 Sep 2020</h5>
 <blockquote>
 <!--
 \setmainfont[Ligatures=TeX]TeX Gyre Pagella \setmathfontTeX Gyre Pagella Math
-
+ 
                                                      __ n=3                  n-k k
-                                          B(t)     = ❯     \ P  \binomnk(1-t)   t
-                                              cubic  ‾‾ k=0
+                                          B(t)     = ❯     \ P  \binomnk(1-t)   t
```
Comment (Pomax) [inline docs/news/2020-09-18.html:154]: Ah, please remember to set your code editor to trim trailing whitespace. Or fix these by hand, but that's probably way more work.

**Review by Pomax — CHANGES_REQUESTED** [review]

**Comment by Pomax** [comment]
Just one small thing and then this can be merged in.

File: docs/news/2020-09-18.html (hunk 2 above)
Comment (EatChangmyeong) [reply docs/news/2020-09-18.html:154]: I believe these files are autogenerated by `npm start`. Do I have to check all of these `html` files manually? I use VS Code and have `"files.trimTrailingWhitespace": true` in my settings file so I can Ctrl+S them away anyway.

**Review by Pomax — APPROVED** [review]

**Comment by Pomax** [comment]
in it goes!

**Pull request merged**